main.js

```diff
@@ -52,10 +55,9 @@ function getLastTheme() {
 function subscribeToRoom(client, roomName) {
     client.subscribe('/rooms/' + roomName, function (message) {
         addToScreen(message.name, message.text, message.timeString);
-        console.log('works');
```

Comment:
ha! hadn't noticed that this console.log had snuck in. Well done on removing it.